data_to_pinecone_llm/README.md

```diff
@@ -0,0 +1,248 @@
+
+### With Terraform
+
+Airbyte allows you to create connectors for sources and destinations, facilitating data synchronization between various platforms. In this project, we're harnessing the power of Terraform to automate the creation of these connectors and the connections between them. Here's how you can set it up:
+
+1. **Navigate to the Airbyte Configuration Directory**:
+
+   Change to the relevant directory containing the Terraform configuration for Airbyte:
+
+   ```bash
+   cd infra/airbyte
+   ```
+
+2. **Modify Configuration Files**:
+
+   Within the `infra/airbyte` directory, you'll find three crucial Terraform files:
+
+   - `provider.tf`: Defines the Airbyte provider.
+   - `main.tf`: Contains the main configuration for creating Airbyte resources.
+   - `variables.tf`: Holds various variables, including credentials.
+
+   Adjust the configurations in these files to suit your project's needs. Specifically, provide credentials for your Notion, BigQuery, and Pinecone connectors. You can utilize the `variables.tf` file to manage these credentials.
+
+3. **Initialize Terraform**:
+
+   This step prepares Terraform to create the resources defined in your configuration files.
+
+   ```bash
+   terraform init
+   ```
+
+4. **Review the Plan**:
+
+   Before applying any changes, review the plan to understand what Terraform will do.
+
+   ```bash
+   terraform plan
+   ```
+
+5. **Apply Configuration**:
+
+   After reviewing and confirming the plan, apply the Terraform configurations to create the necessary Airbyte resources.
+
+   ```bash
+   terraform apply
+   ```
+
+6. **Verify in Airbyte UI**:
+
+   Once Terraform completes its tasks, navigate to the Airbyte UI. Here, you should see your source and destination connectors, as well as the connection between them, set up and ready to go.
+
+## 3. Setting Up the dbt Project
+
+[dbt (data build tool)](https://www.getdbt.com/) allows you to transform your data by writing, documenting, and executing SQL workflows. Setting up the dbt project requires specifying connection details for your data platform, in this case, BigQuery. Here’s a step-by-step guide to help you set this up:
+
+1. **Navigate to the dbt Project Directory**:
+
+   Change to the directory containing the dbt configuration:
+
+   ```bash
+   cd dbt_project
+   ```
+
+2. **Update Connection Details**:
+
+   You'll find a `profiles.yml` file within the directory. This file contains configurations for dbt to connect with your data platform. Update this file with your BigQuery connection details.
+
+3. **Utilize Environment Variables (Optional but Recommended)**:
+
+   To keep your credentials secure, you can leverage environment variables. An example is provided as the `keyfile` config in `profiles.yml`.
```

Comment:
Can you extend this a bit? I'm guessing I need to put the bgiquery credentials in a file an set the env variable to the path of the file, but specifics would be nice to follow along mindlessly

Reply:
I just merged my changes in from the other PR (#85).

The profiles.yml now references env vars, and the env vars are expectd in `.env`, which can be created by copying from `.env.template`.